Geometry/CaloTopology/src/CaloTowerTopology.cc

```diff
@@ -1,46 +1,124 @@
 #include "DataFormats/CaloTowers/interface/CaloTowerDetId.h"
 #include "Geometry/CaloTopology/interface/CaloTowerTopology.h"
+#include <assert.h>
+
+CaloTowerTopology::CaloTowerTopology(const HcalTopology * topology) : hcaltopo(topology) {
+
+  //get number of towers in each hcal subdet from hcaltopo
+  int nEtaHB_, nEtaHE_, nEtaHO_, nEtaHF_;
```

Comment:
@kpedro88 - if you are using trailing underscore to distinguish private data members, these local variables should have not have trailing underscore in their names.